GhostNetwork.Profiles.Api/Controllers/FriendController.cs

```diff
@@ -0,0 +1,142 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.Threading.Tasks;
+using GhostNetwork.Profiles.FriendsFuntionality;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Swashbuckle.AspNetCore.Filters;
+
+namespace GhostNetwork.Profiles.Api.Controllers
+{
+    [Route("[controller]")]
+    [ApiController]
+    public class FriendController : ControllerBase
+    {
+        private readonly IFriendsFunctionalityService friendService;
+
+        public FriendController(IFriendsFunctionalityService friendService)
+        {
+            this.friendService = friendService;
+        }
+
+        /// <summary>
+        /// Search friends by userId
+        /// </summary>
+        /// <param name="skip">Skip friends up to a specified position</param>
+        /// <param name="take">Take friends up to a specified position</param>
+        /// <param name="userId">Filters friends by userId</param>
+        /// <response code="200">Return friends</response>
+        [HttpGet("{userId}")]
```

Comment:
I can be discussed but I think more RESTful style will be [GET] /users/{userId}/friends